tests/test_microsim_model.py

```diff
@@ -27,7 +31,8 @@
 # like `test_step()`.
 @pytest.fixture()
 def test_microsim():
-    population_init = PopulationInitialisation(**population_init_args)
+    with pytest.warns(UserWarning):  # (PopInit should throw a warning because the QUANT api is in test mode
```

Comment:
Think you've removed the test_mode so this change maybe unnecessary

Reply:
Well spotted, removed